### PR TITLE
fix: phone enroll screen

### PR DESCRIPTION
**Pull request by anipendakur-okta**
## PR Checklist

- [X] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [X] Added e2e tests

- [X] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
image
image

### Reviewers:
@haishengwu-okta 

### Issue:

- [OKTA-300141](https://oktainc.atlassian.net/browse/OKTA-300141)

### CODE REVIEW

File: src/v2/view-builder/views/phone/EnrollAuthenticatorPhoneView.js [hunk 1]
```diff
@@ -1,173 +1,43 @@
-import { _, loc, Model } from 'okta';
```
Comment (anipendakur-okta) [inline src/v2/view-builder/views/phone/EnrollAuthenticatorPhoneView.js:1]: Moved this to `EnrollAuthenticatorDataPhoneView` and using this for the `Enter Code` purpose now.

File: src/v2/view-builder/views/phone/EnrollAuthenticatorDataPhoneView.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
+import { _, loc, Model } from 'okta';
```
Comment (anipendakur-okta) [inline src/v2/view-builder/views/phone/EnrollAuthenticatorDataPhoneView.js:1]: This is just moved from `EnrollAuthenticatorPhoneView`, no changes in this file.

File: src/v2/view-builder/views/phone/EnrollAuthenticatorPhoneView.js [hunk 3]
```diff
@@ -1,173 +1,43 @@
...
-    BaseForm.prototype.render.apply(this, arguments);
-    const selectedMethod = this.model.get('authenticator.methodType');
-    const phoneField = this.el.querySelector('.phone-authenticator-enroll__phone');
-    const extensionField = this.el.querySelector('.phone-authenticator-enroll__phone-ext');
-
-    if (selectedMethod === 'voice') {
-      if (!phoneField.classList.contains('phone-authenticator-enroll__phone--small')) {
-        phoneField.classList.add('phone-authenticator-enroll__phone--small');
+    handleResendLink () {
+      this.options.appState.trigger('invokeAction', 'currentAuthenticator-resend');
```
Comment (haishengwu-okta) [inline src/v2/view-builder/views/phone/EnrollAuthenticatorPhoneView.js:14]: can create properties in `ResendView` named `resendActionName` hence only need to override this properties.

Comment (anipendakur-okta) [reply src/v2/view-builder/views/phone/EnrollAuthenticatorPhoneView.js]: updated

**Pull request merged**